src/components/CommentCard.jsx

```diff
@@ -1,3 +1,75 @@
+import { useState,useContext} from "react";
+import { UserContext } from "../App";
+import CommentAdder from "./CommentAdder";
+import {deleteData} from "../utilities/API";
+
+const CommentCard = ({comments,setComments,article_id,showNewCommentForm}) => {
+
+    const {user} = useContext(UserContext);
+    const [errMsg,setErrMsg] = useState("")
+    const [deleteControl,setDeleteControl] = useState(false);
+    const [isDisplay,setIsDisplay] = useState("")
+
+    const handleOnclick = (e) => {
+  
+       const  commentId = Number(e.target.value);
+       const  delUrl = `/comments/${commentId}`;
```

Comment:
we don't necessarily need this variable in here - the path for this will always be this so we could have the path within your request function and just pass down the commentId